src/Locations/ui/GenericLocation.tsx

```diff
@@ -97,10 +95,10 @@ export function GenericLocation({ loc }: IProps): React.ReactElement {
     <>
       <Button onClick={() => Router.toPage(Page.City)}>Return to World</Button>
       <Typography variant="h4" sx={{ mt: 1 }}>
-        {backdoorInstalled && !Settings.DisableTextEffects ? (
+        {backdoorInstalled ? (
           <Tooltip title={`Backdoor installed on ${loc.name}.`}>
```

Comment:
Only loosely related, but this should be:

```suggestion
          <Tooltip title={`Backdoor installed on ${serverMeta.hostname}.`}>
```